### DISCUSSION TITLE
3.15.2 is official?

### DISCUSSION THREAD

**Discussion by okiwan** [encode/django-rest-framework#9588] [Question & Answer]
Quick one: I can see 3.15.2 tag created with some PRs but it is not released. What is the plan for releasing?

**Comment by browniebroke** [comment]
There is a tag in git which you use to see the difference with 3.15.1: https://github.com/encode/django-rest-framework/compare/3.15.1...3.15.2 and it's released on PyPI: https://pypi.org/project/djangorestframework/3.15.2/

Do you mean that it doesn't have a GitHub release?

**Reply by gegoune** [reply]
Seems like latest release on pypi's is dated 19th of June 2024. Has it been updated 'in-place' (I am not sure how pypi handles it)?

**Reply by browniebroke** [reply]
No it hasn't. PyPI does not support updating releases, once published, it's immutable (or at least it should be). The release was done in June and the git tag reflect that: it points to commits from June. 

The [GitHub release](https://github.com/encode/django-rest-framework/releases/tag/3.15.2) was "created" a couple of days ago, but it's associated with [the git tag](https://github.com/encode/django-rest-framework/tags) from June. Anyway, that doesn't change what's on PyPI.

**Reply by gegoune** [reply]
That makes total sense. Didn't pick up the fact that recently created GitHub release is not 'new', but just catching up with pypi release. Thank you for explanation! Happy new year!

**Comment by okiwan** [comment]
Exactly, at least it is reported like that in the index of the repo.

**Reply by browniebroke** [reply]
Was fixed just now: https://github.com/encode/django-rest-framework/releases/tag/3.15.2
